R/init.R

```diff
@@ -1,7 +1,8 @@
 if (interactive() &&
   Sys.getenv("RSTUDIO") == "" &&
   Sys.getenv("TERM_PROGRAM") == "vscode") {
-  if (requireNamespace("jsonlite", quietly = TRUE)) local({
+  if (requireNamespace("jsonlite", quietly = TRUE) &&
+      requireNamespace("rlang", quietly = TRUE)) local({
```

Comment:
Do we need to throw a warning to alarm user that we need the two packages (if they are not installed). Otherwise, users may get confused.

Reply:
We already have the warning at the bottom of `init.R` but I haven't updated it yet.